debug/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/launchConfigurations/LaunchConfigurationFilteredTree.java

```diff
@@ -67,6 +67,7 @@ public LaunchConfigurationFilteredTree(Composite parent, int treeStyle, PatternF
 		fFilters = filters;
 		fPatternFilter = filter;
 		fTreeStyle = treeStyle;
+		filter.setIncludeLeadingWildcard(true);
```

Comment:
I'm torn where is the best place to set it: is it here or in the same class as where PatternFilter is constructed? Anyway, it's really nitpicking and probably not profitable to debate this for a 1-liner that would be easy to change later if necessary.